src/Insurello.AsyncExtra/AsyncExtra.fs

```diff
@@ -75,14 +75,14 @@ module AsyncResult =
                     | Error err1, Error _ -> Error err1
             }
 
-    let traverse : ('a -> 'b) -> List<AsyncResult<'a, 'err>> -> AsyncResult<'b list, 'err> =
+    let traverse : ('a -> AsyncResult<'b, 'err>) -> 'a list -> AsyncResult<'b list, 'err> =
         fun transformer list ->
```

Comment:
@lydell @jamil7 @stoft @Mousaka 
I would love to hear your thoughts on this. I've named the function `transformer` since it's what it is. However, in the other functions, we call it `f`. I'm wondering if I should change the name of this one or if I should change the names of the other functions.

Reply:
I have no opinion

Reply:
Or just leave it as is.

Reply:
I usually refrain from calling functions what they are supposed to be. The type signature usually tells more, but I don't think it's a deal breaker. However, you might consider changing it just to keep consistency across the code base, or change all the other `f`s to have names :)